src/lib.rs

```diff
@@ -609,112 +720,4 @@ pub mod pallet {
-
-        /// Convert a native address to a Move address.
-        pub fn to_move_address(address: &T::AccountId) -> Result<AccountAddress, Error<T>> {
-            let address = AccountId32::decode(&mut address.encode().as_ref())
-                .map_err(|_| Error::InvalidAccountSize)?;
-
-            let account_bytes: [u8; 32] = address.into();
-            Ok(AccountAddress::new(account_bytes))
-        }
-
-        /// Publish the module using the appropriate gas strategy.
-        pub fn raw_publish_module(
-            address: &T::AccountId,
-            bytecode: Vec<u8>,
-            gas: GasStrategy,
-        ) -> Result<VmResult, Error<T>> {
-            let storage = Self::move_vm_storage();
-
-            let vm = Mvm::new(storage, BalanceAdapter::new())
-                .map_err(|_err| Error::<T>::PublishModuleFailed)?;
-            let address = Self::to_move_address(address)?;
-
-            let result = vm.publish_module(&bytecode, address, gas);
-
-            Ok(result)
-        }
-
-        /// Publish the bundle using the appropriate gas strategy.
-        pub fn raw_publish_bundle(
-            address: &T::AccountId,
-            bundle: Vec<u8>,
-            gas: GasStrategy,
-        ) -> Result<VmResult, Error<T>> {
-            let storage = Self::move_vm_storage();
-
-            let vm = Mvm::new(storage, BalanceAdapter::new())
-                .map_err(|_err| Error::<T>::PublishBundleFailed)?;
-            let address = Self::to_move_address(address)?;
-
-            let result = vm.publish_module_bundle(&bundle, address, gas);
-
-            Ok(result)
-        }
-
-        pub fn get_module_abi(
-            address: &T::AccountId,
-            name: &str,
-        ) -> Result<Option<ModuleAbi>, Vec<u8>> {
-            let vm = Self::move_vm()?;
-
-            // TODO: Return a normal message to the RPC caller
-            let address = Self::to_move_address(address).map_err(|_| vec![])?;
-
-            vm.get_module_abi(address, name)
-                .map_err(|e| format!("error in get_module_abi: {:?}", e).into())
-        }
-
-        pub fn get_module(address: &T::AccountId, name: &str) -> Result<Option<Vec<u8>>, Vec<u8>> {
-            let vm = Self::move_vm()?;
-
-            // TODO: Return a normal message to the RPC caller
-            let address = Self::to_move_address(address).map_err(|_| vec![])?;
-
-            vm.get_module(address, name)
-                .map_err(|e| format!("error in get_module: {:?}", e).into())
-        }
-
-        pub fn get_resource(
-            account: &T::AccountId,
-            tag: &[u8],
-        ) -> Result<Option<Vec<u8>>, Vec<u8>> {
-            let vm = Self::move_vm()?;
-            // TODO: Return a normal message to the RPC caller
-            let address = Self::to_move_address(account).map_err(|_| vec![])?;
-
-            vm.get_resource(&address, tag)
-                .map_err(|e| format!("error in get_resource: {:?}", e).into())
-        }
-    }
```

Comment:
Don't look at the red change block above, see more info [here](https://github.com/eigerco/pallet-move/pull/116/files#r1451510322)